librz/main/rz-bin.c

```diff
@@ -1070,38 +1164,56 @@ RZ_API int rz_main_rz_bin(int argc, const char **argv) {
 				sdb_query(bin->cur->sdb, query);
 			}
 		}
-		rz_core_file_close(fh);
-		rz_core_fini(&core);
-		return 0;
+		result = 0;
+		goto err;
 	}
 #define isradjson (rad == RZ_MODE_JSON && actions > 0)
```

Comment:
Rename this one please as well.

Reply:
This is existing code i haven't touch, let it be.